test/test_skymax.py

```diff
@@ -340,17 +340,23 @@ def test_filtering(self):
                                 low_frequency_cutoff=self.low_freq_filter,
                                 normalized=False)
         hpc_corr_R = real(hpc_corr)
-        I_plus, corr_plus, n_plus = matched_filter_core\
-            (hplus, stilde, psd=self.psd,
-             low_frequency_cutoff=self.low_freq_filter, h_norm=1.)
+        I_plus, _, n_plus = matched_filter_core(
+            hplus,
+            stilde,
+            psd=self.psd,
+            low_frequency_cutoff=self.low_freq_filter,
+            h_norm=1.
+        )
         # FIXME: Remove the deepcopies before merging with master
-        I_plus = copy.deepcopy(I_plus)
-        corr_plus = copy.deepcopy(corr_plus)
-        I_cross, corr_cross, n_cross = matched_filter_core\
-            (hcross, stilde, psd=self.psd,
-             low_frequency_cutoff=self.low_freq_filter, h_norm=1.)
-        I_cross = copy.deepcopy(I_cross)
-        corr_cross = copy.deepcopy(corr_cross)
+        I_plus = copy.deepcopy(I_plus).astype(numpy.complex64)
```

Comment:
If you're doing `astype` it will deepcopy anyway if the dtypes don't match ... so this would copy *twice*.

Reply:
Yes. I think this is fine because it happens in a test, not in critical code. Also, those arrays are double precision regardless of Numpy's version, I think the precision is just what the HDF files were created with. So I can simply remove the deepcopy if you agree.